### PR TITLE
[windows] updated v5.4.0

### PR DESCRIPTION
**Pull request by seanyen**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by seanyen** [comment]
@conda-forge/suitesparse This is a version bump and ready for review and merge. Thanks!

**Comment by jschueller** [comment]
please rebase from master, it's 5.7.2 now

**Comment by seanyen** [comment]
@jschueller 

It looks like the SuiteSparse for Windows build lives for its own on this separate branch since [this](https://github.com/conda-forge/suitesparse-feedstock/pull/30) and it takes the source from this project instead: https://github.com/jlblancoc/suitesparse-metis-for-windows, which attempts to keep CMake recipe for cross-platform compiling and now it is ported to `v5.4.0`.

I don't think doing a simple rebase of this feedstock can take it to `v5.7.2`. Or maybe I misread your comments? What's the step I should take?

**Comment by jschueller** [comment]
oh, right, I did not see that.

**Comment by seanyen** [comment]
@conda-forge/suitesparse @conda-forge/core This is ready for review and merge. Thanks!

**Comment by beckermr** [comment]
pls only bump the team and not core on these. We can help if this is not merged for about a week or so.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,36 +9,30 @@ package:
...
     - {{ compiler('cxx') }}
   host:
     - blas 1.1 {{ variant }}
     - openblas
-    - tbb
-    - metis
 
   run:
     - blas 1.1 {{ variant }}
     - openblas
```
Comment (isuruf) [inline recipe/meta.yaml:35]: This needs to be changed to the blas scheme mentioned in the docs

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,36 +9,30 @@ package:
...
+  sha256: 1e1a901e62ac368c8c525bd43cf900e2e1133d2b20a58d5254561bb585af9a9c  # [win]
   patches:
     - gh_56.patch  # [win]
 
 build:
-  skip: true  # [win32 or unix]
-  number: 202
+  number: 0
   features:
     - blas_{{ variant }}
```
Comment (isuruf) [inline recipe/meta.yaml:20]: Remove these 2 lines.

File: recipe/meta.yaml (hunk 1 above)
Comment (seanyen) [reply recipe/meta.yaml:35]: @isuruf Are you talking about this? https://conda-forge.readthedocs.io/en/latest/blas.html Is there any existing PR as the example how to properly add blas as dependency? I did read the doc, however I am still scratching my head how to do it right in `conda-build` recipe.

**Comment by seanyen** [comment]
@conda-forge-admin, please rerender

**Comment by seanyen** [comment]
@isuruf Btw, I saw there are some warnings like these:

Do you recommend to add `libopenblas` to the run requirements? Currently I am only adding `blas` meta-package as the run dependency.

```
WARNING (suitesparse,Library\bin\cholmod.dll): Needed DSO Library\bin\openblas.dll found in ['libopenblas']
WARNING (suitesparse,Library\bin\cholmod.dll): .. but ['libopenblas'] not in reqs/run, (i.e. it is overlinking) (likely) or a missing dependency (less likely)
```

**Comment by seanyen** [comment]
@isuruf hmm, the CI seems not to like the changes. What should we do for the next step?

**Comment by isuruf** [comment]
Try and delete `%PREFIX%/Library/lib/cmake/` ?

**Comment by seanyen** [comment]
@isuruf Just wondering, what's the latest doc for how to integrate with `blas`? Is this one still updated to dated? https://conda-forge.readthedocs.io/en/latest/blas.html

**Comment by isuruf** [comment]
It's the one at https://conda-forge.org/docs/maintainer/knowledge_base.html#blas

**Comment by seanyen** [comment]
I see, so I probably looked at a old site (https://conda-forge.readthedocs.io/) instead of the latest one (https://conda-forge.org/docs/index.html).

**Comment by isuruf** [comment]
Thanks. I'll see if I can get that old site deleted.

**Comment by seanyen** [comment]
@conda-forge/suitesparse Friendly ping. I think this is ready for merge. Thanks.

**Comment by h-vetinari** [comment]
Great to see this happening. I was planning to get around to doing this myself after having build suitesparse for windows within conda-forge/cvxopt-feedstock#42.

I detailed some of that in #51 - I believe (hope?) that metis is not necessary and that we can go to 5.7.2 directly.

**Comment by seanyen** [comment]
> Great to see this happening. I was planning to get around to doing this myself after having built suitesparse for windows within [conda-forge/cvxopt-feedstock#42](https://github.com/conda-forge/cvxopt-feedstock/pull/42).

Happy to see the progress on this too. For my short-term goal, I just want to have `ceres` Windows port with `suitesparse` enabled, so I can do a `conda-forge` feedstock for `https://github.com/cartographer-project/cartographer`. And I am happy to see any attempts to get it converged toward to that direction. :)

**Comment by grlee77** [comment]
I am inclined to go ahead and merge this soon given that it is now passing CI. That said, I don't have experience with Windows + SuitSparse/METIS and would appreciate any feedback from other maintainers.

**Comment by h-vetinari** [comment]
@seanyen True, there's no reason not to merge this, any progress can come on top of this PR.

**Comment by seanyen** [comment]
Thank you all!

**Pull request merged**